config/config-example.js

```diff
@@ -3,6 +3,48 @@
 // The server port - the port to run Pokemon Showdown under
 exports.port = 8000;
 
+// The server address - the address at which Pokemon Showdown should be hosting
+//   This should be kept set to 0.0.0.0 unless you know what you're doing.
+exports.bindaddress = '0.0.0.0';
+
+// workers - the number of networking child processes to spawn
+//   This should be no greater than the number of threads available on your
+//   server's CPU. If you're not sure how many you have, you can check from a
+//   terminal by running:
+//
+//   $ node -e "console.log(require('os').cpus().length)"
+//
+//   Using more workers than there are available threads will cause performance
+//   issues. Keeping a couple threads available for use for OS-related work and
+//   other PS processes will likely give you the best performance, if your
+//   server's CPU is capable of multithreading. If you don't know what any of
+//   this means or you are unfamiliar with PS' networking code, leave this set
+//   to 1.
+exports.workers = 1;
+
+// ssl - SSL key and certificate configuration for HTTPS serving
+//   At the moment, no servers other than smogon.psim.us can be set up to serve
+//   over SSL. For the sake of any possibility of user-run servers becoming
+//   capable of serving over HTTPS in the future, the shape of the
+//   configuration object should look like this:
+//
+//   exports.ssl = {
+//		port: 443,
+//		options: {
+//			key: require('fs').readFileSync('/path/to/privkey.pem'),
+//			cert: require('fs').readFileSync('/path/to/fullchain.pem'),
+//			// Other configuration options for HTTPS servers. Check the
+//			// documentation for Node's https module for more information.
+//			// These will need to be configured to use absolute paths rather
+//			// than buffers of the files' contents in the future.
+//		},
+//	}
+//
+//  Unless you're Zarel or are hosting your own client on your own, this has no
```

Comment:
@Zarel is this accurate?

Reply:
Actually it may be a good idea to make `Config.ssl` use absolute paths now rather than warning about that change but not doing anything about it until later so there isn't so much reconfiguration needed once the Go rewrite can finally get merged

Reply:
we should really wait until it's at least possible for third-party servers to be HTTPS before adding this one in. The others are fine, though.

Reply:
Alright. Is it fine to just keep it in like `// exports.ssl = null;` with no comment, just so we don't forget about it?